File: frontend/gamingDate/lib/model/game.dart
class Game {
  int _gameId;
  String _gameName;
  String _gameLogoUrl;
  DateTime _startTime;
  DateTime _endTime;
  int _rating;

  Game(this._gameId, this._gameName, this._gameLogoUrl, this._startTime,
      this._endTime, this._rating);

  int get gameId => _gameId;

  String get gameName => _gameName;

  int get rating => _rating;

  DateTime get endTime => _endTime;

  DateTime get startTime => _startTime;

  String get gameLogoUrl => _gameLogoUrl;
}

File: backend/gamingdate/src/main/java/com/gamingdate/classes/Game.java
package com.gamingdate.classes;

import java.sql.Timestamp;

import lombok.Data;
import lombok.NonNull;

@Data
public class Game {
    @NonNull Long gameId;
    @NonNull String gameName;
    String gameLogoURL;
    @NonNull Timestamp startTime;
    @NonNull Timestamp endTime;
    Integer rating;
}
